### PR TITLE
FLW-454 Not updated available balance when confirmed transaction

### PR DESCRIPTION
**Pull request by ef1rspb**

### CODE REVIEW

File: CommonWallet/Core/Classes/Modules/TransferConfirmation/TransferConfirmationPresenter.swift [hunk 1]
```diff
@@ -57,6 +70,90 @@ final class TransferConfirmationPresenter {
...
+            deliverOn: .main,
+            executing: changesBlock,
+            failing: failBlock,
+            options: options
+        )
+    }
+
+    private func validateTransfer(completion: (Result<Void, Error>) -> Void) {
+        guard let balances = balances, let metadata = metadata else {
+            // balances & metadata had been validated at previous screen
```
Comment (ArsenyZ) [inline CommonWallet/Core/Classes/Modules/TransferConfirmation/TransferConfirmationPresenter.swift:128]: Aren't they always validated on the first screen?

Comment (ef1rspb) [reply CommonWallet/Core/Classes/Modules/TransferConfirmation/TransferConfirmationPresenter.swift:128]: You're right, but the problem is that user balance could change before transfer confirmation. Scenario:
1. Open Transfer
2. Proceed to Confirmation
3. Perform transfer with another app (i.e. polkadot js)
4. Back to fearless: Tap on `Confirm transfer` -> fails, but fees were paid (

Comment (ArsenyZ) [reply CommonWallet/Core/Classes/Modules/TransferConfirmation/TransferConfirmationPresenter.swift:128]: Sure, i now this issue, but i don't quite understand this success completion

Comment (ERussel) [reply CommonWallet/Core/Classes/Modules/TransferConfirmation/TransferConfirmationPresenter.swift:128]: Yep, mostly likely due to poor connection the values might be present but those are from the cache and there is no difference with the check from the previous screen. Data provider works in a way that it first returns data from the cache and then makes a request to the backend.

**Pull request closed without merging**